lr-cli/cortx_setup/commands/config/get.py

```diff
@@ -50,6 +50,11 @@ def run(self, key=None):
         try:
             val_fr_conf = Conf.get(index, key)
             self.logger.debug(f"Config value for '{key}' : {val_fr_conf}")
+            if not val_fr_conf:
+                self.logger.warning(
+                   f"Given key '{key}' is possibly not found "

```

Comment:
Change this message:
`Configuration key {key} does not exist`

Reply:
Updated